content_scripts/desktopAdapter.js

```diff
@@ -375,7 +375,11 @@ createNameSpace('realityEditor.device.desktopAdapter');
 
                     // emit an event if this is a newly "discovered" matrix
                     if ((!object.matrix || object.matrix.length !== 16) && msgData.newValue.length === 16) {
-                        callbackHandler.triggerCallbacks('objectMatrixDiscovered', {objectKey: msgData.objectKey});
+                        // TODO callbackHandler is probably a figment of this code's imagination
+                        if (typeof callbackHandler !== 'undefined') {
+                            // eslint-disable-next-line
+                            callbackHandler.triggerCallbacks('objectMatrixDiscovered', {objectKey: msgData.objectKey});
+                        }
```

Comment:
Going through the git history, looks like I removed the callback handler in c06e61ff8c11f8c9b15043cae42aa5f77b9a0696 when I was cleaning up a lot of the old code from the Unity remote rendering days, but I missed this use of callbackHandler – I think it's safe to remove it entirely now.

Reply:
Nice, thanks!